src/views/ProjectView.vue

```diff
@@ -131,8 +131,19 @@ const sessionNameExists = (name) => {
 	return uniqueDataSessions.value.some(session => session.name === name)
 }
 
+const saveLargeImages = async (data) => {
+	const largeImages = await data.results
+	store.commit('setLargeImages', largeImages)
+}
+
+const loadLargeImages = async (option) => {
```

Comment:
Just an idea - we could have the store maintain the logic for retrieving a large image from source. I.e. move this function into the store as an action it can do. Then have a function in the store that is get a large (or small) image given its basename. The store would look in its cache, and if not present retrieve it an put it in the cache. This way multiple components (such as the wizard, datasession view, and project view) could all just call the same store function to get the images they want to display, and the code to retrieve and cache those images would be reused within the store.